src/calendar/tests/unit/Calendar.spec.tsx

```diff
@@ -881,6 +884,8 @@ registerSuite('Calendar with min-max', {
 					.setProperty('@date-picker', 'minDate', minDate)
 					.setProperty('@date-picker', 'maxDate', maxDate)
 			);
+
+			restore();
```

Comment:
we shouldn't restore stubs in tests if possible (instead favour before/after's), as if this test errors anywhere in between the stub and this restore, it likely breaks all the following tests also.